src/packages/context/context.py

```diff
@@ -10,22 +10,38 @@
 
 See LICENSE.md
 """
-from typing import Optional, Tuple, List, TYPE_CHECKING
+import typing
```

Comment:
[Question] Why is this imported wholesale instead of simply adding `ClassVar` to the imports below?

Reply:
In this instance, we ended up importing an excessive number of names from this module.

Stylistically it is better after about 5 names to just import the module wholesale.

Further, I argue its more readable to have `typing.ClassVar` than `ClassVar` etc.

Reply:
I will fix the other type hints in this module to conform to the change as well, while im looking at the file.